### ISSUE TITLE
Reconfigure Stolon's "deadKeeperRemovalInterval

### ISSUE THREAD

**Issue by davissp14** [fly-apps/postgres-ha#34]
Stolon will hold onto dead keepers for up to 48 hours, by default.  Each standby keeper has an associated replication slot that holds information about the standby's current WAL position.  The replication slot basically tells Postgres not to recycle or remove any WAL files that could ultimately prevent it from catching up with the leader.  That being said, we should aim to remove dead keepers on a shorter interval to help prevent needless retention of WAL files.  

There are other configuration tweaks that we can make to help with this as well, but i'll address this in a separate issue.

**Comment by anacrolix** [comment]
I've run in to this bug, is there a workaround? Can I edit the config to make it something shorter, or run a command to tell stolon to let go of keepers?

**Comment by davissp14** [comment]
@anacrolix You currently have two options:

1. You can manually remove the keeper.
```
  fly ssh console --app <app-name>
  export $(cat /data/.env | xargs)
  
  # Identify the keeper you'd like to remove
  stolonctl status
  
  # Remove the keeper
  stolonctl: `stolonctl failkeeper <keeper-id>`
```

2. You can adjust the `deadKeeperRemovalInterval` setting:
```
  fly ssh console --app <app-name>
  export $(cat /data/.env | xargs)
  
  stolonctl  update --patch '{ "deadKeeperRemovalInterval" : "12h" }'
```

Reference: https://github.com/sorintlab/stolon/blob/master/doc/cluster_spec.md

**Comment by anacrolix** [comment]
I think I used `removekeeper` for some reason, which did the trick.

**Comment by davissp14** [comment]
@anacrolix Sorry, that's what I mean't.  😅

**Comment by davissp14** [comment]
Seems like there may be cases where deadKeepers are not being removed on the expected interval: https://community.fly.io/t/no-keeper-available-runaway-wal/7153/9

**Comment by davissp14** [comment]
This issue has been resolved with release `v0.0.29`:
https://github.com/fly-apps/postgres-ha/releases/tag/v0.0.29

The default `deadKeeperRemovalInterval` has been lowered from 48h to `1h` with release:
https://github.com/fly-apps/postgres-ha/releases/tag/v0.0.30


The new default `deadKeeperRemovalInterval` interval will only be applied to new apps, existing apps can adjust the interval by doing patching their stolonctl config:

```
  fly ssh console --app <app-name>
  stolonctl  update --patch '{ "deadKeeperRemovalInterval" : "1h" }'
```
